lib/jsonapi/query_parser.ex

```diff
@@ -194,4 +201,6 @@ defmodule JSONAPI.QueryParser do
     view = Keyword.fetch!(opts, :view)
     struct(JSONAPI.Config, opts: opts, view: view)
   end
+
+  defp atomize_map(map), do: for {key, val} <- map, into: %{}, do: {String.to_atom(key), val}
```

Comment:
We do not want to do this. Atoms are not garbage collected. The reason I did this with map.get was to only pull off the keys I needed before setting them in the struct.

Reply:
Sure, but, correct me if i'm wrong, we're talking about 4 additional atoms here (atoms are unique so `:page` counts once).  
Reference:
https://medium.com/everydayhero-engineering/elixir-atoms-3b0f06bc47ef

I'm just atomizing the keys in the map for ease of converting to a struct.

Reply:
Actually, I suppose I see the issue since this doesn't limit query parameters prior to atomization. Let me make a change to this.